internal/manager/config/config.go

```diff
@@ -129,6 +129,10 @@ const (
 	// rather than use the embedded UI.
 	CustomUILocation = "custom_ui_location"
 
+	// Gallery Cover Regex
+	GalleryCoverRegex        = "gallery_cover_regex"
+	galleryCoverRegexDefault = `(poster|cover|folder|board)(\.jpg|\.jpeg|\.png)$`
```

Comment:
The default regex should probably match any file extension instead of just those three - maybe something like `(poster|cover|folder|board)\.[^\.]+$`.

Reply:
I'm not really a gallery user but aren't they only importing image extensions listed in your `Settings > Library > Media content extensions`? This pretty much ensures that all the extensions will be image related unless the user manually adds something not supported.